indra/newview/llpuppetmotion.cpp

```diff
@@ -212,11 +202,14 @@ LLMotion::LLMotionInitStatus LLPuppetMotion::onInitialize(LLCharacter *character
         ids.insert(CHEST_ID);
         mIKSolver.setSubBaseIds(ids);
 
-        //// HACK: white list of sub-roots.
-        //// This HACK prevents the spine from being included in the IK solution,
-        //// effectively preventing the spine from moving.
+        //// TEMPORARY HACK: whitelist of sub-roots.
+        //// This HACK forces joints to form the base of their chain
+        //// effectively preventing them from being moved during the IK solution.
+        //// The plan is to disable this when we have a more robust full-avatar IK solution.
         //ids.clear();
         //ids.insert(CHEST_ID);
+        //ids.insert(COLLAR_LEFT_ID);
+        //ids.insert(COLLAR_RIGHT_ID);
         //mIKSolver.setSubRootIds(ids);
```

Comment:
While testing changes and trying new things I briefly tried whitelisting the collar bones out of the IK chains.  In the end I decided to keep them enabled, but I'm leaving this experimental code around in case it is needed later.